backend/app.js

```diff
@@ -1,16 +1,61 @@
-import express from 'express';
-import path from 'path';
-import logger from 'morgan';
+import express from "express";
+import graphqlHTTP from "express-graphql";
+import { buildSchema } from "graphql";
+import logger from "morgan";
+import path from "path";
+import {
+	createArchipelago,
+	createIsland,
+	createIslander,
+} from "./helpers/mutation_resolvers";
+import {
+	fetchArchipelagoInfo,
+	fetchArchipelagos,
+	fetchIslanderInfo,
+	fetchIslanders,
+	fetchIslandInfo,
+	fetchIslands,
+} from "./helpers/query_resolvers";
```

Comment:
Might be more consistent to stick with camelCase for file names. Or are we wanting to go with snake_case? I'm okay with either, just want to be consistent!

Reply:
Old habits, yeah I'll rename my files without camelcase, I don't mind it.